packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -57,6 +57,12 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       label: whiteSpacePreWrap;
       white-space: pre-wrap;
     `,
+    context: css`
```

Comment:
What about moving this into `getLogRowStyles`? That way we could reference it in the
```
&:hover {
        .${context} {
          visibility: visible;
```
https://github.com/grafana/grafana/pull/20485/files#diff-78c9dedf93895f2b6eca9af99c05d682R64
and we would remove implicit need to have `show-context` class just for the hover behaviour.